core/Azure.Mcp.Core/tests/Azure.Mcp.Tests/Client/RecordedCommandTestsBase.cs

```diff
@@ -84,17 +98,22 @@ public virtual void RegisterVariable(string name, string value)
     }
 
     // used to resolve a recording "path" given an invoking test
-    private static readonly RecordingPathResolver _pathResolver = new();
+    protected static readonly RecordingPathResolver _pathResolver = new();
 
     protected virtual bool IsAsync => false;
 
     // todo: use this when we have versioned tests to run this against.
     protected virtual string? VersionQualifier => null;
 
+    protected override async ValueTask LoadSettingsAsync()
```

Comment:
Any reason to override with just a call to base?

Reply:
Yeah. Because if I call base.LoadSettingsAsync() like I _did_ ,there is no way to disable the default behavior.

If I make it an overridden function, then override _that_ in recordedCommandTestHarness, everything works the way I expect!